app/src/app/navtitle/navtitle.component.sass

```diff
@@ -37,14 +37,30 @@ h2
   top: 3px
 
   @media (max-width: $breakpoint-tablet)
-    top: -3px
     font-size: 9pt
     font-weight: 800
 
 .long-title
   @media (max-width: $breakpoint-tablet)
     display: none
 
+mat-menu .nav-link
+    @media (min-width: $breakpoint-tablet)
+      display: none
```

Comment:
This does not work. The content of the menu is moved from the mat-menu to the overlay (out of reach) when the user opens the menu. Hence it is not possible to have CSS selectors that "cross" the menu limit. From my experiment, the solution here is to add a div in the menu with an id, and to use this id to detect that the .nav-link is in the menu.